### PR TITLE
LPS-178022 Cleanup SearchResult for optional removal

### PR DESCRIPTION
**Pull request by julschong**

### CODE REVIEW

**Comment by julschong** [comment]
ci:test:sf

**Comment by julschong** [comment]
ci:test:search

**Comment by Kvale1733** [comment]
@julschong No regressions found.

File: modules/apps/portal-search/portal-search-web/src/main/java/com/liferay/portal/search/web/internal/portlet/SearchPortletSearchResultPreferences.java [hunk 1]
```diff
@@ -41,8 +40,8 @@ public SearchPortletSearchResultPreferences(
 	}
 
 	@Override
-	public Optional<String> getFieldsToDisplayOptional() {
-		return Optional.empty();
+	public String getFieldsToDisplay() {
+		return StringPool.BLANK;
```
Comment (tinatian) [inline modules/apps/portal-search/portal-search-web/src/main/java/com/liferay/portal/search/web/internal/portlet/SearchPortletSearchResultPreferences.java:44]: @julschong Please move this change to last commit, this is not a usage you need to apply to.

**Comment by julschong** [comment]
no new changes, only moved file changes between commits

**Comment by julschong** [comment]
ci:test:sf

**Comment by tinatian** [comment]
See https://github.com/brianchandotcom/liferay-portal/pull/133942

**Pull request closed without merging**